pkg/cwapi/signals.go

```diff
@@ -57,6 +49,11 @@ func (ctx *ApiCtx) pushSignals() error {
 			return fmt.Errorf("api push signal: return bad HTTP code (%d): %s", resp.StatusCode, string(body))
 		}
 	}
+
+	if resp.StatusCode != 401 && (jsonResp.Message == "" || jsonResp.Message != "OK" || jsonResp.StatusCode != 200) {
```

Comment:
why ignore 401 ?

Reply:
Because `401` mean the token expired. We want to renew it instead of returning an error